notebooks/EDA Climate.ipynb

```diff
@@ -0,0 +1,1821 @@
+{
```

Comment:
<p>Boomer Jesse can't see the white contours on gray. Anyway I think the plot is too busy with the contours. If you want something fancy, plot the marginals on the margins, but the main plot should just be a scatterplot imo.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/laos-climate-change/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>